### ISSUE TITLE
Verbose option for commands

### ISSUE THREAD

**Issue by ResidentBiscuit** [moovweb/gvm#173]
Compiling Go on a micro computer like a Pi can take awhile. Being able to have verbose output as to what's going on with the process would be much appreciated.

**Comment by vsaw** [comment]
@ResidentBiscuit Which version of Go where you installing on your Pi? I tried with 1.5 but did not get it working (see https://github.com/moovweb/gvm/issues/175) How did your installation go?

**Comment by ResidentBiscuit** [comment]
I installed 1.5.2, didn't seem to have any issues. I can post some more info when I get home.

**Comment by masoodkamyab** [comment]
Have you learned anything about software programming?

Dude, every command-line interface has a verbose mode. please change your profession, for example, make some food for people, and DO NOT try software programming anymore.

God made you for another purpose.

**Comment by aeshakhzod** [comment]
Agree. This feature is essential not only when you compile it on slow machines, but also if your network connection is unstable and you don't know what is exactly happening.
